frontend/src/scenes/instance/SystemStatus/systemStatusLogic.ts

```diff
@@ -17,6 +18,10 @@ export const systemStatusLogic = kea<systemStatusLogicType<SystemStatus>>({
             [] as SystemStatus[],
             {
                 loadSystemStatus: async () => {
+                    const { user } = userLogic.values
+                    if (user?.is_multi_tenancy && !user.is_staff) {
+                        return []
+                    }
                     return (await api.get('_system_status')).results
```

Comment:
Why this change?

If we want to limit access it would make more sense to do this at the api level.

Reply:
This is what fixes #2541. We already have this limitation in the backend (https://github.com/PostHog/posthog/blob/bf506e3d0b8038c164b70842385040d5037f641e/posthog/views.py#L49), however because we were calling the endpoint, we were showing an unnecessary error.